routes/api/category-routes.js

```diff
@@ -0,0 +1,77 @@
+const router = require('express').Router();
+const { Category, Product } = require('../../models');
+
+// The `/api/categories` endpoint
+
+router.get('/', async (req, res) => {
+  try {
+    const categoryData = await Category.findAll();
```

Comment:
consider adding logic for some additional cases here e.g. if there are no categories present e.g.
```  
  if (!categoryData.length) {
      return res.status(404).json({
        error: "No categories found"
      });
    }
```